src/app/content/search/components/SearchResultsSidebar/SearchResultContainers.tsx

```diff
@@ -89,6 +94,7 @@ const SearchResult = (props: {
           page={props.page}
           result={thisResult}
           scrollTarget={target}
+          queryParams={{query: props.currentQuery}}
           onClick={() => props.selectResult(thisResult)}
```

Comment:
@TomWoodward to answer your question this handler is used to close mobile search results when one is selected.